src/main/java/org/ozwillo/dcimporter/controller/PublikAPIController.java

```diff
@@ -19,34 +19,30 @@
 public class PublikAPIController {
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(PublikAPIController.class);
-	
+
 	@Value("${publik.datacore.project}")
-    private String datacoreProject;
-	
+	private String datacoreProject;
+
 	@Autowired
 	PublikService publikService;
-	
+
 	private final DatacoreClient datacoreClient;
-    private final SystemUserService systemUserService;
-	
-    
-    @Autowired
-    public PublikAPIController(DatacoreClient datacoreClient, SystemUserService systemUserService) {
-        this.datacoreClient = datacoreClient;
-        this.systemUserService = systemUserService;
-    }
-    
+	private final SystemUserService systemUserService;
+
+	@Autowired
+	public PublikAPIController(DatacoreClient datacoreClient, SystemUserService systemUserService) {
+		this.datacoreClient = datacoreClient;
+		this.systemUserService = systemUserService;
+	}
+
 	@RequestMapping(value = "/form/", method = RequestMethod.POST)
-    public void getForm(@RequestBody FormModel form) {
-		
-		LOGGER.debug("formModel --> :"+form.toString());
-		
-		DCResource dcResource = publikService.convertToDCResource(form);
-		LOGGER.debug("DCResource --> "+dcResource.toString());
-		systemUserService.runAs(() ->
-        datacoreClient.saveResource(datacoreProject, dcResource)
-		);
+	public void getForm(@RequestBody FormModel form) {
```

Comment:
What in case of an error saving the form into the DC ? What can we send back to Publik ? Is Publik able to retry sending the form ?